### ISSUE TITLE
IdsDate Picker: A11Y: Calendar is not properly accessible through screen reader.

### ISSUE THREAD

**Issue by CindyMercadoReyes** [infor-design/enterprise-wc#1031]
**Describe the bug**
Calendar is not properly accessible through screen reader.

WCAG Guideline: 2.1.1 Keyboard

**To Reproduce**

1. Launch the URL: https://main.wc.design.infor.com/ids-date-picker/example.html
2. Navigate to date widget using arrow keys and verify
3. Calendar is not properly accessible through screen reader. While navigating inside calendar screen reader remain silent and does not read the controls.

**Expected behavior**
Calendar should be properly accessible through screen reader.

User Impact: If calendar will not be accessible then screen reader user will not be able to access its functionality.

**Screenshots**
https://www.screencast.com/t/lXnaxipSq

**Platform**

 - Windows 10
 - Browser Name: Chrome

**Additional context**
Add any other context about the problem here. None
